src/shortcuts/exportContext.js

```diff
@@ -15,13 +15,22 @@ const Icon = ({ fill = 'black', size = 20, style }) => <svg version="1.1" classN
 export default {
   id: 'exportContext',
   name: 'Export Context',
-  description: 'Export the current context as plaintext',
+  description: 'Export the current context as plaintext or html',
   svg: Icon,
-  exec: () => {
+  exec: (e, type) => {
```

Comment:
This `exec` function should now simply dispatch `{ type: 'showModal', id: 'export' }`. The downloading itself should be in the `ExportModal` component.